src/Brahma.FSharp.OpenCL.Translator/QuotationTransformers/TransformMinMax.fs

```diff
@@ -0,0 +1,42 @@
+namespace Brahma.FSharp.OpenCL.Translator.QuotationTransformers
```

Comment:
Why so complex? Why we can not use direct mapping to OpenCL ```min``` and ```max```?

Reply:
#130